tests/snapshots/sample.modified.rs

```diff
@@ -10,7 +10,7 @@ fn main() {
         "The perimeter of the rectangle is {} pixels.",
         perimeter(&rect1)
     );
-    println!(r#"This line contains invalid utf8:  "�"#;
```

Comment:
What's going on in this changeset?

Reply:
My IDE probably did something to that when I re-saved the file after adding the tabs section. I'll attempt to fix that.